packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -876,6 +876,17 @@ export default class Editor implements IEditor {
         });
     }
 
+    /**
+     * Remove a Content Edit feature.
+     * @param feature The feature to remove
+     */
+    public removeContentEditFeature(feature: GenericContentEditFeature<PluginEvent>) {
+        const core = this.getCore();
+        feature?.keys.forEach(key => {
+            core.edit.features[key] = [];

```

Comment:
no, do not set to empty array because there may be other features in this array.

You need to find the index of the given feature and use splice() function to remove it.

Then if it end up with an empty array, delete this key from core.editor.features.